src/core/a-entity.js

```diff
@@ -423,15 +425,15 @@ var proto = Object.create(ANode.prototype, {
   },
 
   removeComponent: {
-    value: function (name) {
+    value: function (name, force) {
```

Comment:
Why do you need to remove `default` components? Cannot you store and restore the initial values? The flag breaks the component precedence model and It will make things confusing and introduce weird edge cases and bugs.